env0/utils.go

```diff
@@ -50,6 +50,29 @@ func stringInSlice(str string, strs []string) bool {
 	return false
 }
 
+func reasourceDataGetValue(fieldName string, d *schema.ResourceData) interface{} {
+	dval := d.Get(fieldName)
+	if dval == nil {
+		return nil
+	}
+
+	_, okInt := dval.(int)
+	_, okBool := dval.(bool)
+	_, okString := dval.(string)
+	if okString || okBool || okInt {
+		//lint:ignore SA1019 reason: https://github.com/hashicorp/terraform-plugin-sdk/issues/817
+		if _, exists := d.GetOkExists(fieldName); !exists {
```

Comment:
If it's an int, boolean or string: use GetOkExists.
(GetOk would return false, also if the values are zeroes).

This will allow "zero" values of int, boolean or string to be written to the interface if they're explicitly configured.

E.g.
```
some_resource some_resource_name {
    some_value = false
}
```